src/boot/efi/meson.build

```diff
@@ -349,8 +349,7 @@ foreach archspec : efi_archspecs
 endforeach
 
 foreach efi_elf_binary : efi_elf_binaries
-        # FIXME: Use build_tgt.name() with meson >= 0.54.0
-        name = fs.name(efi_elf_binary.full_path()).split('.')[0]
+        name = efi_elf_binary.name()
```

Comment:
hmm, the commit msg doesn't really match what this code does?

Reply:
It does, no?

Reply:
`typeof(efi_elf_binary) == build_tgt`

Reply:
ah